build/cdc/dev/Dockerfile.nl-python.cdc.dev

```diff
@@ -13,8 +13,14 @@
 # limitations under the License.
 
 # Development Dockerfile for Custom Data Commons NL server
+# Stage 1: Download base dc model from GCS.
+FROM google/cloud-sdk:slim as model-downloader
 
-# Use official Python runtime as a parent image
+# Download NL embeddings model
+RUN mkdir -p /tmp/datcom-nl-models \
+    && gsutil -m cp -R gs://datcom-nl-models/ft_final_v20230717230459.all-MiniLM-L6-v2/ /tmp/datcom-nl-models/
```

Comment:
optional: You can [download the embeddings](https://github.com/datacommonsorg/website/blob/587c43c0928a8f7248908c365b21675e3010be8e/build/cdc_services/Dockerfile#L121) as well.

Reply:
Added! Thank you